enterprise/backend/src/metabase_enterprise/serialization/load.clj

```diff
@@ -35,25 +35,19 @@
    [metabase.util.date-2 :as u.date]
    [metabase.util.i18n :refer [trs]]
    [metabase.util.log :as log]
-   [toucan.db :as db]
-   [yaml.core :as yaml]
-   [yaml.reader :as y.reader])
+   [metabase.util.yaml :as yaml]
+   [toucan.db :as db])
   (:import
-   (java.time.temporal Temporal)
    (java.util UUID)))
 
 (set! *warn-on-reflection* true)
 
-(extend-type Temporal y.reader/YAMLReader
```

Comment:
I played around with this, and it doesn't seem like this was ever actually called in the first place.